raiden-dapp/src/locales/en.json

```diff
@@ -1,28 +1,28 @@
 {
   "address-input": {
     "input": {
-      "placeholder": "Enter an address or ens name..."
+      "placeholder": "Enter an address or ENS name..."
     },
     "blockie-alt": "Selected token address blockie",
     "error": {
-      "invalid-address": "The input doesn't seem like a valid address or ens name",
-      "no-checksum": "The address is not in checksum format",
-      "empty": "The address cannot be empty",
-      "ens-resolve-failed": "Could not resolve an address for {url}"
+      "invalid-address": "Please enter a valid address or ENS name",
+      "no-checksum": "Please enter an address in checksum format. You can convert your address on https://ethsum.netlify.com",
+      "empty": "Please enter an address or ENS name",
+      "ens-resolve-failed": "Please enter a resolvable ENS name"
     }
   },
   "amount-input": {
     "input": {
-      "hint": "Please enter the amount you wish to deposit into this channel."
+      "hint": "Please enter an amount"
     },
     "error": {
-      "empty": "The amount cannot be empty",
-      "too-many-decimals": "The token supports only up to {decimals} decimals",
+      "empty": "Please enter an amount",
+      "too-many-decimals": "Please use only up to {decimals} decimals for this token",
```

Comment:
Starting with `Please use...` sounds more like a suggestion, that you could somehow avoid. Here we have a hard limitation, isn't there a better way to emphasize that fact?

Reply:
I think the `only` makes this a hard limit

Reply:
Not really, if I say to someone "Please use only up to x of y" it is up to their discretion to use up to `x`. It might mean that there is a limited supply in total, but it doesn't make it a per person hard limit.

Reply:
Which wording do you suggest then?

Reply:
I cannot think of a proper wording at the moment.

Reply:
Let's ask @christianbrb if he has a better idea